client/my-sites/stats/site.jsx

```diff
@@ -326,6 +326,7 @@ class StatsSite extends Component {
 			isJetpack,
 			isSitePrivate,
 			isOdysseyStats,
+			isOdyssey,
```

Comment:
This is a bit confusing having both `isOdyssey` and `isOdysseyStats` here. Is this the case where one represents API routing? If so, we should probably rename it now to avoid confusion.

Reply:
> If so, we should probably rename it now to avoid confusion.

I think so. That would be the refactoring work, as we will tackle them all at once later. Context: p1733887222490759-slack-C82FZ5T4G.

Reply:
My bad, maybe `isWPAdmin` a better name for `isOdyssey`?

Reply:
Addressed in [c0df942](https://github.com/Automattic/wp-calypso/pull/97658/commits/c0df94262ee18b88c097f76cef5fa8b84079908c).